app/assets/stylesheets/finder_frontend.scss

```diff
@@ -397,12 +397,12 @@
       @include core-16;
       vertical-align: baseline;
       border-bottom: solid 1px $border-colour;
-      margin: $gutter-half 0;
+      margin: 0 0 $gutter-half 0 ;
       padding-bottom: $gutter-half;
 
       .result-count {
         @include bold-48;
-        margin-right: $gutter-one-third;
+        margin-right: $gutter-one-third / 2;
```

Comment:
Intentionally not moved across or not _yet_ moved across?
What shall I do here then?